dts/bindings/base/base.yaml

```diff
@@ -19,6 +19,18 @@ properties:
     required: true
     description: compatible strings
 
+  hardware-id:
+    type: string
+    description: Device hardware id
+
+  unique-id:
+    type: string
+    description: Device instance number
+
+  class-code:
+    type: string
+    description: Device class id
```

Comment:
this is not hardware description, and so it should not be in devicetree.

Reply:
No more required and is removed with updated code